### PR TITLE
[stable/minio] Fix configmap issue, add ingress resource and general refactor

### PR DESCRIPTION
**Pull request by rtluckie**
- fix issue caused by ConfigMaps now being mounted ReadOnly
- add ingress resource
- consolidate svc resource to support all deployment modes
- update labels and selectors to align with helm best practices
- general cleanup to align with helm best practices/patterns observed in `helm create`
- update values, README and _helpers accordingly
- bump image tag
- bump chart version

### CODE REVIEW

File: stable/minio/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Distributed object storage server built for cloud applications and devops.
 name: minio
-version: 0.5.7
+version: 0.6.0
```
Comment (unguiculus) [inline stable/minio/Chart.yaml]: Maybe this deserves a 1.0.0?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/minio/Chart.yaml (hunk 1 above)
Comment (rtluckie) [reply stable/minio/Chart.yaml]: @unguiculus SGTM.  Fixed.

**Pull request merged**